### DISCUSSION TITLE
Adding daily meeting

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1501] [Q&A]
Hi!  We have a meeting that meets at the same time every morning.  Is there a way to add that as one meeting that meets 7 times?  Or do I need to create 7 meetings?

Thanks!

**Comment by joshreisner** [comment]
youve got to add it 7 times - sorry

the thread right below this one is a request to [create a "duplicate meeting" button](https://github.com/code4recovery/12-step-meeting-list/discussions/1476) which we will add at some point soon ;)

**Comment by ACAwebbuilder** [comment]
Thanks Josh!  No problem at all.  So grateful for this plug-in.

I missed that request...I didn't think to search on "duplicate"....just searched for "daily".  Thanks for letting me know!
